index.Rmd

```diff
@@ -286,7 +290,7 @@ In order to achieve these objectives, you should try to follow a few good practi
     - Consistent placement of curly braces.
 * [Make your own packages](#__optional__Writing_packages_).
 * Keep a similar [directory structure](../RProjectManagement/index.html) across your projects.
-* Use [version control](../RProjectManagement/index.html) to develop/maintain your projects and packages.
+* Use [version control](../RProjectManagement/index.html#version-control) to develop/maintain your projects and packages.
 * Never place `rm(list = ls())` anywhere in your script. If you do so, you may [find your computer set on fire](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/).
 * Use relative file paths and never use `setwd()`. Do press the button below every time you open a script, and assume everyone else will do that too.
 
```

Comment:
The image below of the demo for setting the working directory is  from a Mac. However, most students will be using Windows version of RStudio. Could be more helpful to have a screenshot of how to set the working directory from Windows. See attachment. 
![RStudio_screenshot](https://github.com/jacottemonroe/Scripting4GeoIntro/assets/141016851/9f6d029b-96b4-4007-b18e-d6b68f5bf2bd)